packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -2,23 +2,25 @@ import 'package:flutter/material.dart';
 import 'package:smooth_app/generic_lib/buttons/smooth_action_button.dart';
 import 'package:smooth_app/generic_lib/design_constants.dart';
 
+/// Custom Dialog to use in the app
 ///
-///	Open by calling
-///
-///showDialog<void>(
+/// showDialog<void>(
```

Comment:
```suggestion
/// ```dart
/// showDialog<void>(
```